docs/gui-colour-palette.md

```diff
@@ -0,0 +1,62 @@
+# Colour Palette
+
+Thank you for thinking to contribute to the look of the Telescope. This is a guide to the colour palette that we use to make our web site looks pretty and up-to-date.
+
+|    #FFFFFF, rgb (255,255,255)     |   #A4D4FF, rgb (164, 212, 255)    |    #3670A5, rgb (54, 112, 165)    |
+| :-------------------------------: | :-------------------------------: | :-------------------------------: |
+| ![](./images/#FFFFFF.png=100x100) | ![](./images/#A4D4FF.png=100x100) | ![](./images/#3670A5.png=100x100) |
+
+|   #9E9E9E, rgb (158, 158, 158)    |     #002944, rgb (0, 41, 68)      |
+| :-------------------------------: | :-------------------------------: |
+| ![](./images/#9E9E9E.png=100x100) | ![](./images/#002944.png=100x100) |
+
+## Header
+
+**Title:**
```

Comment:
Maybe something like this?
![#A4D4FF](https://placehold.it/15/A4D4FF/000000?text=+)  **Title:**

Reply:
@cindyledev oh wow this looks cool. I definitely use your suggestion. Do you think I should just remove the table with colours and just put colour square in front of the element?

Reply:
I tried to pull your branch to see the pictures of each colour but none of the pictures loaded properly. I think this is due to the naming convention for the file `#FFFFFF.PNG`, it probably shouldn't start with `#`.

I personally like the table because I can see all the colours in the same space

Reply:
@cindyledev ok so I'll fix that naming and will add small colour indicator as you suggested above